Code/GraphMol/FileParsers/MultithreadedSmilesMolSupplier.h

```diff
@@ -28,7 +28,7 @@ class RDKIT_FILEPARSERS_EXPORT MultithreadedSmilesMolSupplier
       const Parameters &params = Parameters(),
       const SmilesMolSupplierParams &parseParams = SmilesMolSupplierParams());
   MultithreadedSmilesMolSupplier();
-  ~MultithreadedSmilesMolSupplier() override;
+  virtual ~MultithreadedSmilesMolSupplier() {close();};
```

Comment:
I think the dtor here only needs to be virtual if you are planning on inheriting from this class, which I don't think we are.

Reply:
See previous comment